src/org/mozilla/javascript/NativeCallSite.java

```diff
@@ -133,22 +133,7 @@ private Object js_toString(Scriptable obj)
         return sb.toString();
     }
 
-    private Object getUndefined()
```

Comment:
Thanks! That old code is kind of like something from one of those "don't write code like this" web sites ;-)